### DISCUSSION TITLE
UI bug loading explorer and project pages

### DISCUSSION THREAD

**Discussion by ivankhrenkov777** [subquery/subql#997] [Frontier TestNet - Bug Reporting]
The Projects page displays project data incorrectly (name and image)
![image](https://user-images.githubusercontent.com/94184412/167384864-61455310-f393-4e4e-bf21-08ea2d495c55.png)
When selecting any of the projects, an error occurs - "Failed to load project: Failed to fetch"
![image](https://user-images.githubusercontent.com/94184412/167384938-dfc9127b-62f0-4554-8ace-8529e12998b7.png)
Projects Page does not display any data, it is empty
![image](https://user-images.githubusercontent.com/94184412/167385223-b08eedb0-90ee-4830-9d48-b5a9bee5d3e6.png)

**Comment by jim-counter** [comment]
Looks like it could be a CORS issue when trying to retrieve the project icons from IPFS. Seeing this same error on both local admin and the explorer (which is showing the projects but not their icons).

![image](https://user-images.githubusercontent.com/11335994/167386722-1e3a1dc2-8264-4fca-b97c-1a0673a4d8c6.png)

**Comment by rufatnas23** [comment]
I was thinking that the problem was on my side,  so I tried to unregister the indexing account,  and "Transaction failed" error appeared

P.S the same bugs as above appears on my side too

**Comment by mzxyz** [comment]
This issue resolve and for indexer the coordinator service version needs to upgrade to `0.19.0`. Let me know if anyone still face this issue, thanks
